zoo-python-fastapi/app/services/schema/schema_generate.py

```
@@ -2,7 +2,8 @@
 from datetime import datetime, timezone
 
 import aiofiles  # For async file I/O
-
+from app.config import Config
+from app.metrics import rest_counter
```

Ruff

`app.metrics.rest_counter` imported but unused

Remove unused import: `app.metrics.rest_counter`

(F401)